src/screens/SUE/SueHomeScreen.tsx

```diff
@@ -0,0 +1,127 @@
+import { NavigationProp, RouteProp } from '@react-navigation/native';
+import React, { useContext, useMemo } from 'react';
+import { ScrollView, StyleSheet } from 'react-native';
+
+import {
+  Button,
+  ConnectedImagesCarousel,
+  ListComponent,
+  RegularText,
+  SafeAreaViewFlex,
+  SectionHeader,
+  Wrapper,
+  WrapperVertical
+} from '../../components';
+import { ConfigurationsContext } from '../../ConfigurationsProvider';
+import { useStaticContent } from '../../hooks';
+import { QUERY_TYPES } from '../../queries';
+import { SettingsContext } from '../../SettingsProvider';
+import { ScreenName } from '../../types';
+
+type HomeScreenProps = {
+  navigation: NavigationProp<any>;
+  route: RouteProp<any, any>;
+};
+
+const LIST_NAVIGATION_BUTTON = {
+  BOTTOM: 'bottom',
+  TOP: 'top'
+};
+
+export const SueHomeScreen = ({ navigation }: HomeScreenProps) => {
+  const { appDesignSystem = {} } = useContext(ConfigurationsContext);
+  const { globalSettings } = useContext(SettingsContext);
+  const { sections = {} } = globalSettings;
+  const { staticContentList = {}, sueReportListNavigationButton } = sections;
+  const {
+    staticContentName = 'staticContentList',
+    staticContentListDescription,
+    horizontal = true,
+    showStaticContentList = true,
+    staticContentListTitle
+  } = staticContentList;
+
+  const { data } = useStaticContent({
+    refreshTimeKey: `publicJsonFile-${staticContentName}`,
+    name: staticContentName,
+    type: 'json',
+    skip: !showStaticContentList
+  });
+
+  // function to add customized styles from `globalSettings` to the list items
+  const staticContentListItem = useMemo(() => {
+    if (!data) {
+      return [];
+    }
+
+    let listItem = data;
+
+    if (appDesignSystem?.staticContentList) {
+      listItem = listItem?.map((item: any) => ({
+        ...item,
+        appDesignSystem: appDesignSystem.staticContentList
+      }));
+    }
+
+    return listItem;
+  }, [appDesignSystem, data]);
+
+  return (
+    <SafeAreaViewFlex>
+      <ScrollView>
+        <ConnectedImagesCarousel
+          navigation={navigation}
+          publicJsonFile="sueHomeCarousel"
+          refreshTimeKey="publicJsonFile-sueHomeCarousel"
+        />
+
+        {!!sueReportListNavigationButton &&
+          sueReportListNavigationButton === LIST_NAVIGATION_BUTTON.TOP && (
+            <Wrapper noPaddingBottom>
+              <Button
+                invert
+                onPress={() => navigation.navigate(ScreenName.SueList)}
+                title="Meldungen ansehen"
+              />
+            </Wrapper>
+          )}
+
+        {!!staticContentListTitle && (
+          <WrapperVertical style={styles.noPaddingBottom}>
```

Comment:
i guess we need an additional `noPaddingTop` if there is a `LIST_NAVIGATION_BUTTON.TOP` to not make the space that big between button and section header

Reply:
[8d06d51](https://github.com/smart-village-solutions/smart-village-app-app/pull/1244/commits/8d06d5188a2b570c6b674dbcb7f1306c9ca1e49f)